content/docs/concepts/traffic-management/overview/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Overview and Terminology
+description: Learn about Pilot, Istio's core traffic management component, Envoy proxies, and how they enable service discovery and load balancing.
+weight: 1
+keywords: [traffic-management, pilot, envoy-proxies, service-discovery, load-balancing]
+---
+
+With Istio, you can manage [traffic routing](/docs/concepts/traffic-management/routing)
+and [load balancing](/docs/concepts/traffic-management/overview/#load-balancing)
+for your service mesh without having to update your services. Istio simplifies
+configuration of service-level properties like timeouts and retries, and makes
+it straightforward to set up tasks like staged rollouts with percentage-based
+traffic splits.
+
+Istio's traffic management model relies on the following two components:
+
+-  Pilot, the core traffic management component.
+-  Envoy proxies, which enforce configurations and policies set through Pilot.
+
+These components enable the following high-level tasks:
+
+-  Service discovery
+-  Load balancing
+
+## Pilot: Core traffic management {#pilot}
+
+The following diagram shows the Pilot architecture:
+
+{{< image width="40%"
+    link="./pilot-arch.svg"
```

Comment:
"service discovery and traffic rules" isn't connected to anything

Reply:
It's not meant to, it's the legend for the other connectors.

Reply:
Ok, that wasn't obvious to me from an initial view.

Generally a key looks more like this:

`--- Foo and bar`

(e.g. https://i0.wp.com/www.stephanieevergreen.com/wp-content/uploads/2014/11/DirectlyLabelA.jpg)

Reply:
There is no real convention but I'll move the legend a bit further away from the diagram to avoid confusion.